src/main.rs

```diff
@@ -165,11 +169,52 @@ fn real_main(flags: Flags, config: &Config) -> CliResult<Option<()>> {
         Charset::Utf8 => &UTF8_SYMBOLS,
     };
 
-    print_tree(root, kind, &graph, direction, symbols, flag_all);
+    if flag_duplicates {
+        let dups = find_duplicates(&graph);
+        if dups.len() == 0 {
+            println!("No package name is imported more than once");
```

Comment:
I would lean towards leaving this out personally - things like `cargo build` produce no output if there's nothing to do.